azext_iot/tests/test_iot_dps_int.py

```diff
@@ -163,12 +163,15 @@ def test_dps_enrollment_x509_lifecycle(self):
 
     def test_dps_enrollment_symmetrickey_lifecycle(self):
         enrollment_id = self.create_random_name('enrollment-for-test', length=48)
+        enrollment_id2 = self.create_random_name('enrollment-for-test', length=48)
         attestation_type = AttestationType.symmetricKey.value
         primary_key = 'x3XNu1HeSw93rmtDXduRUZjhqdGbcqR/zloWYiyPUzw='
         secondary_key = 'PahMnOSBblv9CRn5B765iK35jTvnjDUjYP9hKBZa4Ug='
         device_id = self.create_random_name('device-id-for-test', length=48)
         reprovisionPolicy_reprovisionandresetdata = 'reprovisionandresetdata'
         hub_host_name = '{}.azure-devices.net'.format(hub)
+        webhook_url = 'https://www.test.test'
+        api_version = '2018-09-01-preview'
```

Comment:
Maybe we test with the same version of the current embedded DPS SDK? The version is `2019-03-31`

Reply:
Updated